src/components/Appointment/index.js

```diff
@@ -81,6 +81,10 @@ function Appointment(props) {
         />
       )}
 
+      {mode === PENDING && (
+        <div>Loading...</div>
```

Comment:
Maybe this should be a `<span>`? Or maybe we need more design work to make this look good?